application/features/user-unit-tests.feature

```diff
@@ -49,3 +49,12 @@ Feature: User Unit Tests
       And the mfaAllowDisable config parameter is false
     When I add backup codes for that user
     Then I see the user's require_mfa property is yes
+
+# Uncomment the following, if you want to manually test whether the
+# User->registerPwnedPasswordGAEvent() method is working
+# Note: this doesn't do any asserting on your behalf. Go check the results yourself.
+# Also note: Google Analytics fails silently if you use the wrong GA4_API_SECRET
```

Comment:
I'm not sure about using a Behat feature for this. It doesn't use any asserts, so it feels more like a dev tool and less like a test. Plus, it requires the addition of the `local.env` file in the docker-compose.yml `env_file` directive. I find it confusing to have overlapping sources of environment variables in docker compose. Can you do this in a yii command line controller and document it in the README, perhaps?